src/bundles/experiments.js

```diff
@@ -0,0 +1,97 @@
+const ACTIONS = {
+  EXP_TOGGLE: 'EXPERIMENTS_EXP_TOGGLE'
+}
+
+// setup experiments
+const EXPERIMENTS = {
```

Comment:
What if `EXPERIMENTS` obtains IPFS Desktop specific experiments from `window.ipfsDesktop.experiments`?

Reply:
Let's stick with the simplicity of configuring all the experiments in one place for now. We can split them later, if we need to.

Reply:
Agree. This is a nice idea but probably unnecessary overhead at this stage. I prefer to have the logic self-contained if possible 👌

Reply:
Then, we need to _know_ when to show them, it means, if we're inside IPFS Desktop or not.

Reply:
Curious: can you also extend translations from IPFS desktop? Because that would be key for this to work.

Reply:
I see we have a `desktop` variable. Although, the user might not support that experiment due to other reasons, such as the ones I've mentioned above

Reply:
See my PR description above, you can use the `desktop: bool` flag for this and the reducer filters out options based on the context :)

Reply:
Actually the `desktop` variable from the example on the first post is **not** used.

Reply:
@cwaring gotcha!